src/awesome-texmacs.tm

```diff
@@ -1,11 +1,13 @@
 <TeXmacs|1.99.14>
 
-<style|notes>
+<style|browser>
```

Comment:
Why you change this package? This has to be left at `notes`.

Reply:
The change happened without me realizing it, I could start everything from scratch as I do not know why the change is there

Reply:
Maybe you can try to change it on a text editor and open again the file to see if TeXMacs changes it again. Apriori there should be no need for that. You see "Notes" as the style of the document if all is correct.

Reply:
Just close this PR and open another one where you have one commit. We can use these try to see and write down some precise instructions on how to contribute.

Reply:
Ok, I'll do that.